### PR TITLE
Updates | Add Language & Make Contents Less Restrictive 

### PR DESCRIPTION
**Pull request by justin-shipstation**
https://auctane.atlassian.net/browse/PLAT-796

https://auctane.atlassian.net/browse/PLAT-1150

https://auctane.atlassian.net/browse/PLAT-1305

### CODE REVIEW

**Review by jbolton209 — DISMISSED** [review]
lgtm

File: src/internal/carriers/packages/package-item.ts [hunk 1]
```diff
@@ -25,20 +25,23 @@ export class PackageItem implements IPackageItem {
...
-      unitPrice: MonetaryValue[_internal].schema.required(),
+      quantity: Quantity[_internal].schema.optional(),
+      unitPrice: MonetaryValue[_internal].schema.optional(),
     }),
   };
 
-  public readonly sku: string;
-  public readonly identifiers: Identifiers;
+  public readonly sku?: string;
+  public readonly identifiers?: Identifiers;
```
Comment (rkrauskopf) [inline src/internal/carriers/packages/package-item.ts:34]: @justin-shipstation It's not necessary to make these optional. `sku` will be initialized to an empty string, `identifers` will just be an empty object.

File: src/internal/orders/sales-order-identifier.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { hideAndFreeze, Identifiers, Joi, _internal } from "../common";
 
 export abstract class SalesOrderIdentifierBase implements ISalesOrderIdentifier {
   public readonly id: string;
-  public readonly identifiers: Identifiers;
+  public readonly identifiers?: Identifiers;
```
Comment (rkrauskopf) [inline src/internal/orders/sales-order-identifier.ts:7]: @justin-shipstation make this required and just let it initialize to an empty object

File: src/public/carriers/packages/package-item.ts [hunk 3]
```diff
@@ -10,12 +10,12 @@ export interface PackageItem {
   /**
    * The Stock Keeping Unit code
    */
-  readonly sku: string;
+  readonly sku?: string;
```
Comment (rkrauskopf) [inline src/public/carriers/packages/package-item.ts:13]: Required and initialize to empty string, `""`

**Review by rkrauskopf — COMMENTED** [review]
@justin-shipstation Looks good! Just make the internal `identfiers` and `sku`s required and initialize them to empty objects and strings like we discussed and we should be g2g!

**Review by bill-robbins-ss — APPROVED** [review]

**Pull request merged**